2015-07-14 10:44:41,949 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:412] An unhandled error occurred: The repository is already open by another process for writing. See the log for more details.
java.lang.IllegalStateException: The repository is already open by another process for writing
	at org.locationtech.geogig.storage.bdbje.JEObjectDatabase.createDatabase(JEObjectDatabase.java:228) ~[geogig-bdbje-1.0-20141218.040451-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.storage.bdbje.JEObjectDatabase.open(JEObjectDatabase.java:201) ~[geogig-bdbje-1.0-20141218.040451-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.storage.bdbje.JEObjectDatabase_v0_2.open(JEObjectDatabase_v0_2.java:19) ~[geogig-bdbje-1.0-20141218.040451-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.storage.ForwardingObjectDatabase.open(ForwardingObjectDatabase.java:36) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.storage.ForwardingObjectDatabase.open(ForwardingObjectDatabase.java:36) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.repository.Repository.open(Repository.java:108) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.GeoGIG.getRepository(GeoGIG.java:150) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.newGeoGIG(GeogigCLI.java:253) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.loadRepository(GeogigCLI.java:228) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.loadRepository(GeogigCLI.java:223) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.getGeogig(GeogigCLI.java:168) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:520) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:383) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:328) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
Caused by: com.sleepycat.je.EnvironmentLockedException: (JE 5.0.84) /var/lib/geoserver_data/geogig/9726b5d4-95de-43e3-ae9c-9ff59a43a737/.geogig/objects The environment cannot be locked for single writer access. ENV_LOCKED: The je.lck file could not be locked. Environment is invalid and must be closed.
	at com.sleepycat.je.log.FileManager.<init>(FileManager.java:368) ~[je-5.0.84.jar:5.0.84]
	at com.sleepycat.je.dbi.EnvironmentImpl.<init>(EnvironmentImpl.java:483) ~[je-5.0.84.jar:5.0.84]
	at com.sleepycat.je.dbi.EnvironmentImpl.<init>(EnvironmentImpl.java:409) ~[je-5.0.84.jar:5.0.84]
	at com.sleepycat.je.dbi.DbEnvPool.getEnvironment(DbEnvPool.java:178) ~[je-5.0.84.jar:5.0.84]
	at com.sleepycat.je.Environment.makeEnvironmentImpl(Environment.java:252) ~[je-5.0.84.jar:5.0.84]
	at com.sleepycat.je.Environment.<init>(Environment.java:233) ~[je-5.0.84.jar:5.0.84]
	at com.sleepycat.je.Environment.<init>(Environment.java:177) ~[je-5.0.84.jar:5.0.84]
	at org.locationtech.geogig.storage.bdbje.EnvironmentBuilder.get(EnvironmentBuilder.java:147) ~[geogig-bdbje-1.0-20141218.040451-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.storage.bdbje.JEObjectDatabase.createEnvironment(JEObjectDatabase.java:140) ~[geogig-bdbje-1.0-20141218.040451-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.storage.bdbje.JEObjectDatabase.createDatabase(JEObjectDatabase.java:226) ~[geogig-bdbje-1.0-20141218.040451-53.jar:1.0-SNAPSHOT]
	... 13 common frames omitted
2015-07-14 10:45:15,309 INFO [main] o.l.g.r.HttpRemoteRepo [HttpRemoteRepo.java:467] Processed 41 objects. Inserted: 38. Existing: 3. Time: 945.4 ms. Compressed size: 6,758 bytes. Uncompressed size: 36,870 bytes.
2015-07-14 10:45:17,756 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:412] An unhandled error occurred: user.name not found in config. Use geogig config [--global] user.name <your name> to configure it.. See the log for more details.
java.lang.IllegalStateException: user.name not found in config. Use geogig config [--global] user.name <your name> to configure it.
	at com.google.common.base.Preconditions.checkState(Preconditions.java:200) ~[guava-17.0.jar:na]
	at org.locationtech.geogig.api.porcelain.CommitOp.resolveCommitter(CommitOp.java:449) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.porcelain.CommitOp._call(CommitOp.java:238) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.porcelain.CommitOp._call(CommitOp.java:57) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:133) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.porcelain.MergeOp.commit(MergeOp.java:371) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.porcelain.MergeOp._call(MergeOp.java:334) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.porcelain.MergeOp._call(MergeOp.java:51) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:133) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.porcelain.PullOp._call(PullOp.java:260) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.porcelain.PullOp._call(PullOp.java:35) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:133) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.porcelain.Pull.runInternal(Pull.java:102) ~[geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:65) ~[geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:525) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:383) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:328) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
2015-07-14 10:47:00,713 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:412] An unhandled error occurred: Doing a checkout without a clean working tree and index is currently unsupported.. See the log for more details.
org.locationtech.geogig.api.porcelain.CheckoutException: Doing a checkout without a clean working tree and index is currently unsupported.
	at org.locationtech.geogig.api.porcelain.CheckoutOp._call(CheckoutOp.java:290) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.porcelain.CheckoutOp._call(CheckoutOp.java:57) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:133) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.porcelain.PullOp._call(PullOp.java:254) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.porcelain.PullOp._call(PullOp.java:35) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:133) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.porcelain.Pull.runInternal(Pull.java:102) ~[geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:65) ~[geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:525) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:383) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:328) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
2015-07-14 10:55:32,446 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:412] An unhandled error occurred: The repository is already open by another process for writing. See the log for more details.
java.lang.IllegalStateException: The repository is already open by another process for writing
	at org.locationtech.geogig.storage.bdbje.JEObjectDatabase.createDatabase(JEObjectDatabase.java:228) ~[geogig-bdbje-1.0-20141218.040451-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.storage.bdbje.JEObjectDatabase.open(JEObjectDatabase.java:201) ~[geogig-bdbje-1.0-20141218.040451-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.storage.bdbje.JEObjectDatabase_v0_2.open(JEObjectDatabase_v0_2.java:19) ~[geogig-bdbje-1.0-20141218.040451-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.storage.ForwardingObjectDatabase.open(ForwardingObjectDatabase.java:36) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.storage.ForwardingObjectDatabase.open(ForwardingObjectDatabase.java:36) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.repository.Repository.open(Repository.java:108) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.api.GeoGIG.getRepository(GeoGIG.java:150) ~[geogig-core-1.0-20141218.040504-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.newGeoGIG(GeogigCLI.java:253) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.loadRepository(GeogigCLI.java:228) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.loadRepository(GeogigCLI.java:223) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.getGeogig(GeogigCLI.java:168) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:520) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:383) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:328) [geogig-cli-1.0-20141218.040455-53.jar:1.0-SNAPSHOT]
Caused by: com.sleepycat.je.EnvironmentLockedException: (JE 5.0.84) /var/lib/geoserver_data/geogig/9726b5d4-95de-43e3-ae9c-9ff59a43a737/.geogig/objects The environment cannot be locked for single writer access. ENV_LOCKED: The je.lck file could not be locked. Environment is invalid and must be closed.
	at com.sleepycat.je.log.FileManager.<init>(FileManager.java:368) ~[je-5.0.84.jar:5.0.84]
	at com.sleepycat.je.dbi.EnvironmentImpl.<init>(EnvironmentImpl.java:483) ~[je-5.0.84.jar:5.0.84]
	at com.sleepycat.je.dbi.EnvironmentImpl.<init>(EnvironmentImpl.java:409) ~[je-5.0.84.jar:5.0.84]
	at com.sleepycat.je.dbi.DbEnvPool.getEnvironment(DbEnvPool.java:178) ~[je-5.0.84.jar:5.0.84]
	at com.sleepycat.je.Environment.makeEnvironmentImpl(Environment.java:252) ~[je-5.0.84.jar:5.0.84]
	at com.sleepycat.je.Environment.<init>(Environment.java:233) ~[je-5.0.84.jar:5.0.84]
	at com.sleepycat.je.Environment.<init>(Environment.java:177) ~[je-5.0.84.jar:5.0.84]
	at org.locationtech.geogig.storage.bdbje.EnvironmentBuilder.get(EnvironmentBuilder.java:147) ~[geogig-bdbje-1.0-20141218.040451-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.storage.bdbje.JEObjectDatabase.createEnvironment(JEObjectDatabase.java:140) ~[geogig-bdbje-1.0-20141218.040451-53.jar:1.0-SNAPSHOT]
	at org.locationtech.geogig.storage.bdbje.JEObjectDatabase.createDatabase(JEObjectDatabase.java:226) ~[geogig-bdbje-1.0-20141218.040451-53.jar:1.0-SNAPSHOT]
	... 13 common frames omitted
